configure.py

```diff
@@ -1,20 +1,9 @@
 import os, platform, subprocess
 
 def checkCommand(shell):
-    if platform.system() == "Windows":
-        return os.system("where " + shell)
-    else:
-        return os.system("which " + shell)
+    return subprocess.check_call(["which", shell])
 
-def checkAll():
-    print("Checking for valid bash")
-    if checkCommand("bash"):
-        print("bash not found")
-        print("You will want a POSIX compatible environment")
-        exit(1)
-    else:
-        print("bash checked")
-    
+def checkAll():    
     print("Checking for rst2html")
     if checkCommand("rst2html"):
         print("rst2html not found")
```

Comment:
The advice in the following line to use pip3 is inappropriate.  rst2html is called from modules/readme2modulestrings.py, which is python2